### PR TITLE
@sarahscott => News article parent component

### PR DESCRIPTION
**Pull request by kanaabe**
This adds a NewsLayout and NewsSections components to the mix. The NewsLayout gets used by the main Article component to render a full article. 

newsarticle

### CODE REVIEW

File: src/Components/Publishing/News/NewsSections.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    return renderedSections
+  }
+
+  renderByline() {
+    const { article: { authors } } = this.props
+
+    if (authors) {
+      return
+      // Return a byline
```
Comment (kanaabe) [inline src/Components/Publishing/News/NewsSections.tsx]: This is for you!

File: src/Components/Publishing/Sections/ImageCollection.tsx [hunk 2]
```diff
@@ -15,26 +15,29 @@ interface ImageCollectionProps {
   targetHeight?: number
   gutter?: number
   sectionLayout?: SectionLayout
+  articleLayout?: Layout
```
Comment (kanaabe) [inline src/Components/Publishing/Sections/ImageCollection.tsx]: This component represents a section called `image_collection`. It's a group of `Image` and `Artwork` components in a row.

For Newsfeed, we don't want the ability to expand images into a fullscreen viewer. We have the ability to turn that feature off on the `Image` and `Artwork` components individually but we need to pass down this `articleLayout` prop to make that work within an `ImageCollection`.

File: src/Components/Publishing/Sections/ImageCollection.tsx [hunk 3]
```diff
@@ -53,6 +56,7 @@ class ImageCollectionComponent extends React.PureComponent<ImageCollectionProps,
           <Image
             image={image}
             sectionLayout={sectionLayout}
+            layout={articleLayout}
```
Comment (kanaabe) [inline src/Components/Publishing/Sections/ImageCollection.tsx]: This is where we pass it through. `Image` now knows that this has a `layout="news"` so it won't render that `IconExpand`.

File: src/Components/Publishing/Sections/Sections.tsx [hunk 4]
```diff
@@ -109,54 +107,48 @@ export class Sections extends Component<Props, State> {
...
   }
 
   getSection(section, index) {
+    const { article } = this.props
+
     const sections = {
       image_collection: (
         <ImageCollection
           sectionLayout={section.layout}
+          articleLayout={article.layout}
```
Comment (kanaabe) [inline src/Components/Publishing/Sections/Sections.tsx]: This is actually all Prettier changes except this line -- we're now passing down the `articleLayout` again for the other Article types that use this more generic `Sections` component.

File: src/Components/Publishing/__stories__/Article.story.tsx [hunk 5]
```diff
@@ -206,4 +211,7 @@ story.add("Feature", () => {
         relatedArticles={[StandardArticle, VideoArticle]}
       />
     )
-  })
\ No newline at end of file
+  })
+  .add("News Article", () => {
+    return <Article article={NewsArticle} />
+  })
```
Comment (kanaabe) [inline src/Components/Publishing/__stories__/Article.story.tsx]: 🗞

File: src/Components/Publishing/News/NewsSections.tsx [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    authors?: any
+    sections?: any
+  }
+}
+
+interface ContainerProp {
+  type: string
+}
+
+export class NewsSections extends Component<Props, null> {
```
Comment (damassi) [inline src/Components/Publishing/News/NewsSections.tsx]: No need to pass in `null` if not using a `State` interface.

**Comment by sarahscott** [comment]
Looks awesome! Merging!

**Pull request merged**